src/foam/nanos/auth/Address.js

```diff
@@ -670,6 +670,19 @@ foam.CLASS({
             }
             return postalCodeError ? postalCodeError : X.INVALID_POSTAL_CODE;
           }
+        },
+        // Pakistan
+        {
+          args: ['postalCode', 'countryId'],
+          query: 'countryId!="PK"||postalCode~/^(\\s*|\\d{5})$/',
+          errorMessage: 'INVALID_POSTAL_CODE',
+          jsErr: function(X) {
```

Comment:
not sure if this is needed - since your using 'errorMessage' ... the messages can have a local setting, and therefore no need to seemingly call the translationService

Reply:
Hmm so the message, can't account for the per country, so this does make sense to add